flow-typed/npm/bluebird_v3.x.x.js

```diff
@@ -1,194 +0,0 @@
-// flow-typed signature: 4c1347a1bf7c69a614112e551403195f
```

Comment:
`"bluebird": "^3.5.0"` is part of `ern-util` package. Verifying if this file is intentionally deleted?

Reply:
@krunalsshah 
I just removed the flow type definitions of bluebird, not the dependency itself.
There is an invalid type def in the file leading to flow errors. Given that we make ultra limited use of bluebird it's ok if we don't have type definitions for it.